<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>867</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page_main">
       <widget class="QPushButton" name="btn_mainticketing">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>200</y>
          <width>226</width>
          <height>451</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(242, 189, 56);
background-color: rgb(242, 189, 56);
color: rgb(255, 255, 255);
</string>
        </property>
        <property name="text">
         <string>영
화
예
매</string>
        </property>
        <property name="autoRepeatInterval">
         <number>100</number>
        </property>
       </widget>
       <widget class="QLabel" name="logo">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>40</y>
          <width>361</width>
          <height>121</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;C G V&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_ticketlookup">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>200</y>
          <width>251</width>
          <height>451</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(37, 186, 221);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>영
화
예
매
조
회</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_ticket">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_movie">
           <attribute name="title">
            <string>예매</string>
           </attribute>
           <widget class="QTableWidget" name="tableWidget_search">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>190</y>
              <width>881</width>
              <height>461</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(231, 231, 231);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="autoScroll">
             <bool>true</bool>
            </property>
            <property name="autoScrollMargin">
             <number>8</number>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="dragDropOverwriteMode">
             <bool>false</bool>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="textElideMode">
             <enum>Qt::ElideLeft</enum>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="horizontalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <column>
             <property name="text">
              <string>장르</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>영화제목</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>상영시간</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>감독</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>주연1</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>주연2</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>주연3</string>
             </property>
            </column>
           </widget>
           <widget class="QPushButton" name="btn_search">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>750</x>
              <y>100</y>
              <width>150</width>
              <height>73</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>맑은 고딕</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(242, 189, 56);</string>
            </property>
            <property name="text">
             <string>검색</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_ticketing">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>360</x>
              <y>670</y>
              <width>186</width>
              <height>73</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>맑은 고딕</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(242, 189, 56);</string>
            </property>
            <property name="text">
             <string>예매하기</string>
            </property>
           </widget>
           <widget class="QComboBox" name="comboBox_genre">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>120</y>
              <width>128</width>
              <height>30</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="logo_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>682</width>
              <height>100</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>맑은 고딕</family>
              <pointsize>28</pointsize>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;영화 조회/ 예매하기&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QComboBox" name="comboBox_title">
            <property name="geometry">
             <rect>
              <x>190</x>
              <y>120</y>
              <width>204</width>
              <height>30</height>
             </rect>
            </property>
            <item>
             <property name="text">
              <string>영화 전체 검색</string>
             </property>
            </item>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_lookup">
           <attribute name="title">
            <string>예매조회</string>
           </attribute>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>120</y>
              <width>901</width>
              <height>542</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>맑은 고딕</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="title">
             <string>예매 정보 입력</string>
            </property>
            <widget class="QPushButton" name="pushButton_done1">
             <property name="geometry">
              <rect>
               <x>360</x>
               <y>420</y>
               <width>150</width>
               <height>73</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>맑은 고딕</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(242, 189, 56);</string>
             </property>
             <property name="text">
              <string>확인</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>100</x>
               <y>120</y>
               <width>86</width>
               <height>59</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>맑은 고딕</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>이름</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_name1">
             <property name="geometry">
              <rect>
               <x>300</x>
               <y>120</y>
               <width>388</width>
               <height>65</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="label_4">
             <property name="geometry">
              <rect>
               <x>70</x>
               <y>220</y>
               <width>215</width>
               <height>59</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>맑은 고딕</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>휴대폰번호</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_phone1">
             <property name="geometry">
              <rect>
               <x>310</x>
               <y>220</y>
               <width>388</width>
               <height>65</height>
              </rect>
             </property>
            </widget>
           </widget>
           <widget class="QLabel" name="logo_3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>361</width>
              <height>91</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>맑은 고딕</family>
              <pointsize>28</pointsize>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;예매 조회&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_info">
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>-10</y>
          <width>831</width>
          <height>1061</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="title">
         <string>예매 정보 입력</string>
        </property>
        <widget class="QPushButton" name="pushButton_done2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>520</y>
           <width>150</width>
           <height>73</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(242, 189, 56);</string>
         </property>
         <property name="text">
          <string>확인</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>210</y>
           <width>86</width>
           <height>59</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>이름</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_name2">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>210</y>
           <width>388</width>
           <height>65</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>300</y>
           <width>215</width>
           <height>59</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>휴대폰번호</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_phone2">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>300</y>
           <width>388</width>
           <height>65</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
